resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/kubernetes/integrationtest/KubernetesSuite.scala

```diff
@@ -355,8 +355,8 @@ private[spark] class KubernetesSuite extends SparkFunSuite with BeforeAndAfter {
       "--executor-memory", "512m",
       "--executor-cores", "1",
       "--num-executors", "1",
-      "--upload-jars", HELPER_JAR,
-      "--upload-files", TEST_EXISTENCE_FILE.getAbsolutePath,
+      "--jars", HELPER_JAR,
+      "--files", TEST_EXISTENCE_FILE.getAbsolutePath,
       "--class", FILE_EXISTENCE_MAIN_CLASS,
       "--conf", "spark.ui.enabled=false",
```

Comment:
The more I think about it the more I'm not sure what having `--files` with a `local://` path even means. If the file is local and the user wants access to it... if they knew the path to give in the `--files` parameter itself, would they not also be able to just use that path directly from the containers? So I don't think testing `--files` with `local` paths is altogether too relevant, really.